pkg/providers/vsphere/config/ethtool_daemonset.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ethtool
+  namespace: {{.eksaSystemNamespace}}
+spec:
+  selector:
+    matchLabels:
+      name: ethtool
+  template:
+    metadata:
+      labels:
+        name: ethtool
+    spec:
+      containers:
+        - name: ethtool
+          image: {{.kindNodeImage}}
+          command: ["/bin/sh"]
+          args: ["-c", "ethtool -K eth0 tx-udp_tnl-segmentation off && ethtool -K eth0 tx-udp_tnl-csum-segmentation off && echo 'done' && sleep infinity"]
```

Comment:
Do we need to a `readinessProbe` to be able to determine when the container has finished? or maybe we can remove the `sleep infinity` and just rely on the pods being on state `Succeeded`?